### PR TITLE
Du 4/log out

### PR DESCRIPTION
**Pull request by nrios14**
Added new page and link to enterprise docs for the new Support Admin Tool

### CODE REVIEW

**Review by michal-at-travisci — DISMISSED** [review]

**Comment by nrios14** [comment]
@michal-at-travisci 
This is ready for merging.

**Comment by piccadilly-circus** [comment]
Added some details about the auth tokens as well.

File: user/enterprise/support-admin-tool.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Support Admin Tool
+layout: en_enterprise
+
+---
+
+> The tool is only visible to Platform maintainers from TCIE 3.x as **Admin-v2**. The tool must be configured with a list of GitHub handles that allow **admin** access.
```
Comment (stan-travis) [inline user/enterprise/support-admin-tool.md:7]: This option is not available for hosted support and administrators?

**Review by stan-travis — APPROVED** [review]
Agreed to release for Enterprise for now.

**Comment by stan-travis** [comment]
Adding to Enterpise documention

**Pull request merged**